uk.co.compucorp.civicrm.tasksassignments/js/controllers/modal/modalAssignment.js

```diff
@@ -245,7 +252,7 @@ define(['controllers/controllers',
                         continue;
                     }
 
-                    taskType = activity.name ? $filter('filter')($rootScope.cache.taskType.arr, { value: activity.name }, true)[0] : '';
+                    taskType = activity.name ? $filter('filter')($rootScope.cache.taskType.arr, {value: activity.name}, true)[0] : '';
```

Comment:
I believe it's better to have a padding whitespace in object literals, like it was before